extensions/in-memory/assetindex-memory/src/test/java/org/eclipse/dataspaceconnector/metadata/memory/InMemoryAssetIndexTest.java

```diff
@@ -253,6 +253,23 @@ void findAll_withSorting() {
         assertThat(index.queryAssets(spec)).containsAll(assets);
     }
 
+    @Test
+    void deleteById_whenPresent_deletes() {
+        var asset = createAsset("foobar");
```

Comment:
Maybe we can create a separate cleanup story to use Faker for the test classes containing hardcoded items? Many tests related to AssetIndex/Loader already use hardcoded names like foobar, but I didn't want to update them to avoid having too much noise in this PR.